client/src/components/FlashCards.jsx

```diff
@@ -54,15 +55,24 @@ const FlashCards = props => {
       }
       return { x, rot, scale, delay: undefined, config: { friction: 50, tension: down ? 800 : isGone ? 200 : 500 } };
     });
-    if (!down && gone.size === cardData.length)
+    if (!down && gone.size === cardData.length) {
+      setShowQuizScoreDialog(true);
```

Comment:
```suggestion
      if (mode === 'quiz') {
        setShowQuizScoreDialog(true);
      }
```

We only want to show the dialog for the quiz